microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -55,7 +55,7 @@ To create an app reservation:
 <!-- ====================================================================== -->
 ## Package your PWA for the Store
 
-Now that you have your app publishing information, generate a Windows app package for your PWA.
+Now that you have your app publishing information, generate a Windows app package for your PWA by using PWA Builder.
```

Comment:
```suggestion
Now that you've gathered the publishing information about your app, generate a Windows app package for your PWA by using PWA Builder.
```
i'm reading markdown w/o any context, but this phrase read ambiguously, 3 different ways.  Purely considering this sentence w/o context, here's one possible resolution of the meaning.

It could mean: 
Now that you've gotten your app to gather information, ...
that's how I initially parsed (probably mis-parsed) it.